src/modules/deposit/service.ts

```diff
@@ -45,117 +45,64 @@ export class DepositService {
-    }
-
-    let query = this.depositRepository.createQueryBuilder("d");
-    query = query.where("d.depositDate is not null");
-    query = query.andWhere(
-      new Brackets((qb) => {
-        qb.where("d.depositorAddr = :userAddress", {
-          userAddress,
-        }).orWhere("d.recipientAddr = :userAddress", { userAddress });
-      }),
-    );
-
-    if (status) {
-      query = query.andWhere("d.status = :status", { status });
-    }
-
-    query = query.orderBy("d.depositDate", "DESC");
-    query = query.take(limit);
-    query = query.skip(offset);
-
-    const [userDeposits, total] = await query.getManyAndCount();
-
-    return {
-      deposits: userDeposits.map(formatDeposit),
-      pagination: {
-        limit,
-        offset,
-        total,
-      },
-    };
-  }
-
-  public async getDeposits(status: "filled" | "pending", limit = 10, offset = 0) {
-    let deposits: Deposit[] = [];
-    let total = 0;
-
-    if (status === "filled") {
-      [deposits, total] = await this.depositRepository
-        .createQueryBuilder("d")
-        .where("d.status = :status", { status })
-        .andWhere("d.depositDate is not null")
-        .orderBy("d.depositDate", "DESC")
-        .take(limit)
-        .skip(offset)
-        .getManyAndCount();
-    } else if (status === "pending") {
-      // filter out pending deposits older than 1 day because the relayer will ignore such deposits using its fixed lookback
-      [deposits, total] = await this.depositRepository
-        .createQueryBuilder("d")
-        .where("d.status = :status", { status })
-        .andWhere("d.amount > 0")
-        .andWhere("d.depositDate > NOW() - INTERVAL '1 days'")
-        .andWhere(`d.depositRelayerFeePct * :multiplier >= d.suggestedRelayerFeePct`, {
-          multiplier: this.appConfig.values.suggestedFees.deviationBufferMultiplier,
-        })
-        .orderBy("d.depositDate", "DESC")
-        .take(limit)
-        .skip(offset)
-        .getManyAndCount();
-    } else {
-      [deposits, total] = await this.depositRepository
-        .createQueryBuilder("d")
-        .where("d.depositDate is not null")
-        .orderBy("d.depositDate", "DESC")
-        .take(limit)
-        .skip(offset)
-        .getManyAndCount();
-    }
-
-    return {
-      deposits: deposits.map(formatDeposit),
-      pagination: {
-        limit,
-        offset,
-        total,
-      },
-    };
-  }
-
-  public async getDepositsV2(query: GetDepositsV2Query) {
+  public async getDepositsForTxPage(query: Partial<GetDepositsForTxPageQuery>) {
     const limit = parseInt(query.limit ?? "10");
     const offset = parseInt(query.offset ?? "0");
-    let queryBuilder = this.depositRepository.createQueryBuilder("d");
-    queryBuilder = queryBuilder.where("d.depositDate is not null");
 
-    if (query.status) {
-      queryBuilder = queryBuilder.andWhere("d.status = :status", { status: query.status });
-    }
+    let queryBuilder = this.depositRepository.createQueryBuilder("d");
```

Comment:
QQ: what does `d` represent in this case?

Reply:
You can pass in an alias, in this case for the `Deposit` entity

Reply:
That feels like it degrades readability, but if it's used elsewhere sgtm!